set.mm

```diff
@@ -505542,7 +505759,7 @@ codes is an increasing chain (with respect to inclusion).  (Contributed
 
   ${
     $d f i j n u v x y $.
-    $( The set of all valid Goedel formulas.  (Contributed by AV,
+    $( The set of all valid Godel formulas.  (Contributed by AV,
```

Comment:
Ah, so you prefer `Godel` over `Goedel`? :)
Too bad we can't have Gödel!

Reply:
Update "HTML code for accented names:"

Reply:
Indeed we have `G&ouml;del`!

Reply:
Yeah, some day perhaps we'll UTF-8-ize everything but that's a big project so unless/until we do that, `&#xf6;` or `&ouml;` is the way to go.

Reply:
I also would prefer Gödel or at least Goedel, but @digama0 definitions were about "Godel-sets", so for consistency, and because it is in MC's mathbox, I adhered to this terminology.

Reply:
I believe we should use `G&ouml;del` everywhere, that is what will give the correct rendering (although it impairs reading the ASCII source). 
In any case we can handle that separately. For now, keeping it consistent is what matters.